domain/src/Services/Math.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Domain\Services;
+
+class Math
+{
+    private const SCALE = 32;
```

Comment:
ℹ️ Ridiculous amount of decimal places by default, ensuring to always capture them all